src/webview/cluster/app/cluster.tsx

```diff
@@ -16,7 +16,8 @@ import {
     Tooltip,
     Typography
 } from '@material-ui/core';
-
+import clsx from 'clsx';
+import { CardBody, CardFooter } from '@patternfly/react-core';
```

Comment:
@msivasubramaniaan Let's just use material UI and remove the patternfly dependency for now.